### DISCUSSION TITLE
YOLOV5question

### DISCUSSION THREAD

**Discussion by 273541643** [mrdbourke/pytorch-deep-learning#394] [Q&A]
![image](https://user-images.githubusercontent.com/117343039/231085459-d9359da1-3e1a-49e8-a94e-f86be7f6463a.png)

**Comment by 273541643** [comment]
How to solve this problem?

**Comment by mrdbourke** [comment]
Hi @273541643,

Usually when I have the `forward()` `NotImplementedError` it's because I've not defined a `forward()` function in my model.

Or I've messed up the spacing somewhere in the model definition (e.g. it might be 2 spaces instead of 4).

I'd check out both of these things and see what happens.
